native/src/components/TtsContainer.tsx

```diff
@@ -54,11 +56,22 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
   const showSnackbar = useSnackbar()
   const title = sentences[0] || t('nothingToRead')
   const shortTitle = truncate(title, { maxChars: TTS_MAX_TITLE_DISPLAY_CHARS })
-  const { data: voices } = useLoadAsync(Tts.voices)
+  const { data: voices, error, refresh, loading } = useLoadAsync(Tts.voices)
   const isLanguageSupported =
     voices && voices.some(({ language }) => getGenericLanguageCode(language) === getGenericLanguageCode(languageCode))
   const enabled = buildConfig().featureFlags.tts
 
+  useReportError(error)
+
+  useEffect(() => {
+    const voicesMissing = voices?.length === 0 || error !== null
+    if (!loading && voicesMissing && voicesRetries < 2) {
```

Comment:
💭 This looks a bit arbitrary and confusing to me, why do we give it exactly two retries? It seems to me like this is hopefully a temporary thing until we figure out what the actual problem is? Or am I misunderstanding that? Maybe we can document that?